setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+
+required = [
+    "Flask",
+    "Jinja2",
+    "MarkupSafe",
+    "MySQL-python",
+    "Werkzeug",
+    "argparse",
+    "itsdangerous",
+    "requests",
+    "wsgiref",
```

Comment:
hmm, I just copy and paste the whole 'requirements.txt'. I just want to ensure all the packages has been installed correctly before tox run. So, could you give me some advice that which package is what we need or which one isn't?